raiden/api/rest.py

```diff
@@ -638,8 +638,11 @@ def open(
             reveal_timeout=reveal_timeout,
         )
 
+        confirmed_block_identifier = views.state_from_raiden(self.raiden_api.raiden).block_hash
```

Comment:
I see this quite often, maybe might be useful to write a view for it

Reply:
I searched for `views.state_from_raiden(self.raiden_api.raiden).block_hash` and there's only one place which does this specific call, the rest of the calls to `views.state_from_raiden(self.raiden_api.raiden)` are used to get the `chain_state` as it's needed for something else and we use that chain_state to get the block_hash. If the pattern repeats, we can introduce the view function in a different PR.

Reply:
`confirmed_block_identifier = views.state_from_raiden(self.raiden).block_hash` is kind of the same think and exists more often

Reply:
I see your point.. this is done.